doc/kvs_error_code.md

```diff
@@ -0,0 +1,29 @@
+# KVS Error Code
```

Comment:
How about create a chapter in main `README.md` about `Debug and issue address` link to this?

Reply:
Yeah, that's another TODO task to improve the main `README.md`. For now, I'm just adding a little doc here.